docs/configuring-playbook-bridge-hookshot.md

```diff
@@ -23,6 +23,11 @@ Other configuration options are available via the `matrix_hookshot_configuration
 
 Finally, run the playbook (see [installing](installing.md)).
 
+### End-to-bridge endcryption
+
+1. Enable by setting `matrix_hookshot_experimental_encryption_enabled: true`
+
+If the crypto store has become corrupted, reset it by running `ansible-playbook -i inventory/hosts setup.yml -K --tags=reset-hookshot-encryption`.
```

Comment:
```suggestion
You can enable [experimental encryption](https://matrix-org.github.io/matrix-hookshot/latest/advanced/encryption.html) for Hookshot by adding `matrix_hookshot_experimental_encryption_enabled: true` to your configuration (`vars.yml`) and [re-running the playbook](installing.md).

If the crypto store ever becomes corrupted, reset it by running the `reset-hookshot-encryption` tag for this Ansible playbook (e.g. `ansible-playbook -i inventory/hosts setup.yml -K --tags=reset-hookshot-encryption`)
```

Reply:
Thanks, I've added something like that with similar wording.